e2e/bitrise.yml

```diff
@@ -110,8 +110,8 @@ workflows:
 
                 # For Xcode 12 make sure EXCLUDED_ARCHS is set to arm architectures otherwise
                 # the build will fail on lipo due to duplicate architectures.
-                echo "EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x86_64__XCODE_$XC_VER = arm64 arm64e armv7 armv7s armv6 armv8" >> $xcconfig
-                echo 'EXCLUDED_ARCHS = $(inherited) $(EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_$(EFFECTIVE_PLATFORM_SUFFIX)__NATIVE_ARCH_64_BIT_$(NATIVE_ARCH_64_BIT)__XCODE_$(XCODE_VERSION_MAJOR))' >> $xcconfig
+                #echo "EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x86_64__XCODE_$XC_VER = arm64 arm64e armv7 armv7s armv6 armv8" >> $xcconfig
```

Comment:
I mentioned in the PR description that I edited the e2e tests so any reviewer can try this PR in a real scenario with zero effort. This will not be part of the final PR.